OpenRA.Mods.Common/UpdateRules/Rules/RemoveReloadPrefix.cs

```diff
@@ -0,0 +1,73 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class RemoveReloadPrefix : UpdateRule
+	{
+		public override string Name { get { return "Remove ReloadPrefix from With*AimAnimation traits"; } }
+		public override string Description
+		{
+			get
+			{
+				return "ReloadPrefix has been removed from WithAimAnimation and WithTurretAimAnimation.\n"
```

Comment:
Am I mixing something up here or did you just add `WithAimAnimation` in this PR together with an update rule refactoring the new trait?

Reply:
Hm yeah, now that you mention it... guess I should put all the WithAimAnimation stuff in one update rule and let this here only update WithTurretAimAnimation.

Reply:
The simpler the better.  What is the strategy forward here?